kong/conf_loader.lua

```diff
@@ -545,7 +545,7 @@ end
 -- `port` (number), `ssl` (bool), `http2` (bool), `listener` (string, full listener)
 local function parse_listeners(values)
   local list = {}
-  local flags = { "ssl", "http2", "proxy_protocol" }
+  local flags = { "ssl", "http2", "proxy_protocol", "transparent" }
```

Comment:
This would also add the "transparent" option to the http listeners no? shouldn't they be limited to the TCP listeners?

Reply:
We want it available on both kinds. Infact stream support isn't in master/next right now, so this PR is *only* adding it for HTTP.